dashboard/app/config.go

```diff
@@ -204,6 +204,10 @@ type ConfigManager struct {
 	FixBisectionDisabled bool
 	// CC for all bugs that happened only on this manager.
 	CC CCConfig
+	// Other parameters being equal, helps to determine the order of crashes for a bug.
+	// The higher Priority, the higher on top a crash will be.
+	// Priority is an integer in the range [0;9].
```

Comment:
To make a qemu manager lower priority, we will need to assign non-0 priority to all 20+ other managers.
Maybe it will be better if we make this [-3..3] and then shift by 3 when calculating crash priority? Then we can assign priority just to managers we want to penalize/prioritize.

Reply:
Hmm. I just realized that due to repro retesting we gradually update crash priorities, so to avoid the splash of origin retestings we'd rather want to keep as many crash priorities unchanged as we can.

So I've updated the PR to keep the range [-5;5] and not to shift it by 5. If it's negative, the priority will be smaller.

Alternatively, we can replace `Priority` with `PriorityDrop` and consider all our managers to have the highest priority by default.

Reply:
Priority looks better than PriorityDrop b/c we may want to increase priority for one "default" manager (or some set of "known good" managers).